src/smpi/private.h

```diff
@@ -171,15 +171,21 @@ XBT_PRIVATE smx_mailbox_t smpi_process_mailbox();
 XBT_PRIVATE smx_mailbox_t smpi_process_remote_mailbox(int index);
 XBT_PRIVATE smx_mailbox_t smpi_process_mailbox_small();
 XBT_PRIVATE smx_mailbox_t smpi_process_remote_mailbox_small(int index);
+XBT_PRIVATE smx_mailbox_t smpi_process_remote_mailbox_migration(int index);
 XBT_PRIVATE xbt_mutex_t smpi_process_mailboxes_mutex();
 XBT_PRIVATE xbt_mutex_t smpi_process_remote_mailboxes_mutex(int index);
 XBT_PRIVATE xbt_os_timer_t smpi_process_timer();
 XBT_PRIVATE void smpi_process_simulated_start();
-XBT_PRIVATE double smpi_process_simulated_elapsed();
+
+/* 
+ * By rktesser: double smpi_process_simulated_elapsed() and
```

Comment:
This is a commit log, not a comment. Please remove this. (and I feel nervous about exposing stuff to the public API)